app/controllers/monologue/admin/base_controller.rb

```diff
@@ -1,6 +1,6 @@
 class Monologue::Admin::BaseController < Monologue::ApplicationController
   include Monologue::ControllerHelpers::Auth
   force_ssl if Monologue::Config.admin_force_ssl # TODO: find a way to test that with capybara
-  
-  layout "layouts/monologue/admin"
+  layout 'layouts/monologue/admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.